### ISSUE TITLE
jump to selection start marker

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#1969]
**Reported by peter on 2011-11-27 11:09**
Hi all,
Is it possible to give the user a possibility to quickly move review cursor to the position of "start of text to be copyed" (nvda+f9)? This should be useful in many situations, e.g. temporary placemarker replacement in virtual buffers, hopefully workaround for patented fs functionality.

**Comment by leonardder** [comment]
Interesting thought, but this shouldn't be a replacements for placemarkers IMO.

**Comment by Adriani90** [comment]
cc: @feerrenrut
